.env.sample

```diff
@@ -0,0 +1,32 @@
+# Set a unique secret key for the project, required for running outside DEBUG mode
+SECRET_KEY=Ensure_you_set_a_secret_key_this_is_important!
+
+LOCAL_WEB_PORT=5000
+
+# Database details for postgres container
+DB_TYPE=postgresql
```

Comment:
we newer change this

Reply:
Yes, and I think that we need to delete mysql implementation, because we never used this